### DISCUSSION TITLE
iOS open source or not

### DISCUSSION THREAD

**Discussion by ConceptAttached** [darkreader/darkreader#6852] [Q&A]
Seeing some complaints about dark reader snatching keystrokes. Is the new iOS version open source? Is the community watching and checking?

**Comment by alexanderby** [comment]
Hi, where do you see these complaints?

**Reply by ConceptAttached** [reply]
Reviews, I was surprised.

**Reply by ConceptAttached** [reply]
Is iOS version open source or no?

**Reply by bershanskiy** [reply]
No, iOS version is not open source.

**Reply by ConceptAttached** [reply]
> No, iOS version is not open source.

Too bad. Than you for the info.
